web/src/compose_validate.ts

```diff
@@ -44,6 +45,7 @@ export let wildcard_mention_threshold = 15;
 
 export const SPLIT_DELIMITER = "\n\n\n";
 let split_messages_enabled = false;
+let split_messages_banner_shown = false;
```

Comment:
Not sure if a state variable is needed at all... Read through `compose_banners.ts` to see how other banners are handled without a state variable for each

Reply:
I noticed some functions related to `topic_resolved_banner`:
```
export function set_recipient_viewed_topic_resolved_banner(flag: boolean): void {
    recipient_viewed_topic_resolved_banner = flag;
}

export function has_recipient_viewed_topic_resolved_banner(): boolean {
    return recipient_viewed_topic_resolved_banner;
}
```

This is why I added a variable for this. Things will work fine even without this variable, but it is a minor optimization.

Reply:
But `recipient_viewed_topic_resolved_banner` exists to avoid displaying the banner multiple times in a given view; that pattern seems irrelevant here?